src/components/pages/champions/DetailPage.tsx

```diff
@@ -24,10 +26,15 @@ interface DetailPageProps {
 
 export default function DetailPage({ championEnName, queryLane }: DetailPageProps) {
   const router = useRouter();
-  const capitalizedChampionEnName = championEnName.charAt(0).toUpperCase() + championEnName.slice(1);
+  const championId = champions.find(
+    (champion) => champion.enName.toLowerCase() === championEnName.toLowerCase(),
+  )?.championId;
+  const championName = championId
```

Comment:
```suggestion
  const championName = championId !== undefined
```
`championId`가 0일 경우는 없을 거 같긴 한데 혹시 모르니까 명시적으로 처리하죠